Dockerfile.ci

```diff
@@ -6,6 +6,10 @@ ENV BUNDLE_PATH="/home/dependabot/.bundle" \
   BUNDLE_BIN=".bundle/binstubs" \
   PATH=".bundle/binstubs:$PATH:/home/dependabot/.bundle/bin"
 
+RUN pyenv install -s 3.6.13
```

Comment:
Worth a comment on why we've added these for future us?

Reply:
Yeah, good call.

Reply:
:/ I forgot to commit this, will open a separate pr